internal/scorecard/plugins/plugin_runner.go

```diff
@@ -130,9 +132,27 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 		if err != nil {
 			return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("failed to read csv: %v", err)
 		}
+
 		if err = yaml.Unmarshal(yamlSpec, csv); err != nil {
 			return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("error getting ClusterServiceVersion: %v", err)
 		}
+
+		csvValidator := validation.ClusterServiceVersionValidator
+		results := csvValidator.Validate(csv)
+		for _, r := range results {
+			if r.HasError() {
+				var errorMsgs strings.Builder
+				for _, e := range r.Errors {
+					errorMsgs.WriteString(e.Error())
+				}
+				return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("error validating ClusterServiceVersion: %s", errorMsgs.String())
```

Comment:
```suggestion
				return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("error validating ClusterServiceVersion: %s", errorMsgs)
```

Since `%s` invokes `*.String()`.